usr/src/cmd/text-install/osol_install/text_install/welcome.py

```diff
@@ -55,6 +57,15 @@ class WelcomeScreen(BaseScreen):
     def set_actions(self):
         '''Remove the F3_Back Action from the first screen'''
         self.main_win.actions.pop(self.main_win.back_action.key, None)
+        zpool_install_action = Action(curses.KEY_F5, _("InstallToExistingPool"), self.zpool_install)
+        self.main_win.actions[zpool_install_action.key] = zpool_install_action
+
+    def zpool_install(self, dummy):
+        '''We just skip one screen as installer doesn't have means of navigating between
```

Comment:
comment is incorrect - we doesn't skip screen, we set flag and go to next screen. Skipping some screens is done in their show() method if this flag is set